content/deploy/maintain/audit.md

```diff
@@ -0,0 +1,62 @@
+---
+title: 'Archive the PostgreSQL Audit trail'
+date: '2024-03-26T11:20:56-03:00'
+categories: ["how-to"]
+description: How to archive a partition of the Audit trail on your Rhize deployment
+weight: 300
+menu:
+  main:
+    parent: maintain
+    identifier:
+---
+
+The [audit trial]({{< relref "/how-to/audit" >}}) can generate a high volume of data, so it is a good practice to periodically _archive_ portions of it. An archive separates a portion of the data from the database and keeps it for long-term storage. This process involves the use of PostgresQL [Table Partitions](https://www.postgresql.org/docs/current/ddl-partitioning.html).
```

Comment:
Doing this speeds up the query of existing data, whilst allowing for cost effective storage of older data.